### PR TITLE
Fix for duplicate Arktos pod requests

### PR DESCRIPTION
**Pull request by phudtran**
This PR fixes an issue with pod creation and networking for the Arktos integration

### CODE REVIEW

File: mizar/arktos/arktos_service.py [hunk 1]
```diff
@@ -52,17 +55,26 @@ def CreatePod(self, request, context):
         param.body['status']['phase'] = request.phase
         param.body['metadata']['tenant'] = request.tenant
         param.extra = {}
+        ep_name = request.name+"-default-{}-eth0".format(request.tenant)
```
Comment (Hong-Chang) [inline mizar/arktos/arktos_service.py]: "default" and "eth0" may be subjected to be variable?

File: mizar/arktos/arktos_service.py [hunk 2]
```diff
@@ -52,17 +55,26 @@ def CreatePod(self, request, context):
...
+        if ep_opr.store.get_ep(ep_name):
+            logger.info("EP {} already exists! Return OK".format(ep_name))
+            return ReturnCode(
```
Comment (Hong-Chang) [inline mizar/arktos/arktos_service.py]: I think pod create and update are sharing same code. So if user wants to change pod by purpose, such as change its priority, seems to me the change will be unexpectedly ignored.

File: mizar/daemon/interface_service.py [hunk 3]
```diff
@@ -81,13 +85,13 @@ def _CreateVethInterface(self, interface):
         veth_peer = interface.veth.peer
         veth_index = get_iface_index(veth_name, self.iproute)
 
-        if veth_index != -1:
-            self.iproute.link('delete', index=veth_index)
-            veth_index = -1
         if veth_index == -1:
+            logger.info("Creating if {}".format(veth_name))
```
Comment (Hong-Chang) [inline mizar/daemon/interface_service.py]: if => interface

File: mizar/arktos/arktos_service.py (hunk 1 above)
Comment (phudtran) [reply mizar/arktos/arktos_service.py]: Right. The interface part may be anything. However, I'm always seeing that "default". Let me dig into this a bit more.

File: mizar/daemon/interface_service.py (hunk 3 above)
Comment (phudtran) [reply mizar/daemon/interface_service.py]: Fixed

File: mizar/arktos/arktos_service.py (hunk 1 above)
Comment (phudtran) [reply mizar/arktos/arktos_service.py]: Fixed

File: mizar/arktos/arktos_service.py (hunk 2 above)
Comment (phudtran) [reply mizar/arktos/arktos_service.py]: Fixed.

**Review by Hong-Chang — APPROVED** [review]
LGTM assuming the change has been tested.

**Pull request merged**